src/containers/Freelancer/reducer.js

```diff
@@ -0,0 +1,28 @@
+// initial state
+const initialState = {
+  current_freelancer: {}
+}
+
+export default function FreelancerReducer(state = initialState, action) {
+  switch(action.type) {
+    case 'FREELANCER_QUERY': {
+      return {
+        ...state,
+        loading_freelancer: true
+      }
+    }
+
+    case 'FREELANCER_INFO': {
+      return {
+        ...state,
+        loading_freelancer: false,
+        current_freelancer: {
+          ...action.payload.allFreelancers.nodes[0]
```

Comment:
You'll need to change this to only use ``action.payload.node`` if you follow my suggestion about action dispatch above.